### DISCUSSION TITLE
Words repeated in transcriptions obtained with streaming Nova

### DISCUSSION THREAD

**Discussion by davasquez** [deepgram/community#297] [General help] [answered]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

This issue occurs from time to time. For example, if the user says "yes" the transcription sometimes is "Yes." and sometimes is "Yes. Yes." or even "Yes. Yes. Yes."

### If possible, please attach your code or paste it into the text box.

Same code as in the documentation: https://developers.deepgram.com/docs/getting-started-with-live-streaming-audio

**Comment by jjmaldonis** [comment]
Hey @davasquez, does this happen often for you? On occasion our Nova Streaming model will hallucinate. We see this fairly rarely, and our future models should reduce how often this occurs.

**Reply by davasquez** [reply]
Hi @jjmaldonis it happens very often. Actually before, we were using enhanced in spanish and we didn't notice any issue about these repetitions. Then we moved to nova as soon as nova was released for spanish, keeping exactly the same code, the only thing we did was just replacing "enhanced" to "nova" and we started seeing quite a lot of repetitions.

We have also been using nova for english, but we don't currently use it as much as we use for spanish. Nevertheless, we are also seeing repetitions for nova in english.

This is a big issue for us, because sometimes we use deepgram for phone numbers, and if the repetitions occur, then we get the number wrong. Also, in some other usages of deepgram, this issue is getting us into problems.

Probably, this is an issue about some buffering? rather than allucinations? given that it is repeating the same words, instead of allucinating with other words. What is weird is that it happens from time to time, but not always.

**Reply by davasquez** [reply]
Hi @jjmaldonis, any update on this issue? thanks

**Reply by jjmaldonis** [reply]
Hey @davasquez, thanks for the additional information and explaining your specific use cases. That helps me to pass along the urgency internally. We are currently working on new models that should improve this behavior. I don't have an ETA at this time. Would you be able to switch back to the Enhanced models until the new models are available?

**Reply by davasquez** [reply]
Hi @jjmaldonis if you can charge me for the enhanced model with the same price as the nova modal, I can move back to the enhanced model while you fix the issue with nova. I would be very thankful because this issue is getting us into many problems.

**Reply by jjmaldonis** [reply, marked as answer]
Hey @davasquez let's move the convo to email. Please send me an email at jason.maldonis at deepgram dot com.

**Reply by davasquez** [reply]
Hi @jjmaldonis, I sent you an email today. Please contact me. Thanks!

**Comment by LeandroTorresSicilia** [comment]
Hi, @jjmaldonis I have the same issue, I notice it from time to time in my production service. Also, I have a chunk of audio with a test that I run and I can reproduce it almost every time. In my case, it repeats the same word 15 or more times.
Any update on this?

**Comment by kholiavko-roman** [comment]
We have the same issue, unfortunately, we do not have a trace ID for now, will save each id from now to provide it in case we catch this issue again.
